apps/component-examples/src/components/CheckoutsListComponent.tsx

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../jsx.d.ts" />
+import { h } from '../utils/simple-jsx';
+
+export interface CheckoutsListData {
+  subAccountId: string;
+  webComponentToken: string;
+}
+
+export function CheckoutsListComponent(data: CheckoutsListData) {
+  const { subAccountId, webComponentToken } = data;
+
+  return (
+    <div class="two-column-layout">
+      <div class="column-preview">
+        <div class="list-component-wrapper">
+          <justifi-checkouts-list
+            account-id={subAccountId}
+            auth-token={webComponentToken}
+          ></justifi-checkouts-list>
+        </div>
+      </div>
+      <div class="column-output" id="output-pane">
+        <em>Checkouts list output will appear here...</em>
```

Comment:
I dont think we need to have the two-column-layout setup for the examples that render list components. 

If this is intentional, and this was done to show event usage in the list components, then maybe we just need a specific layout designed to show the list component in 75% of the screen and the output-pane in like 25% of the screen.